src/websocket.js

```diff
@@ -182,11 +168,37 @@ const trades = (payload, cb) => {
         tradeId,
       })
     })
-
-    return w
   })
 
-  return () => cache.forEach(w => w.close())
+  return () => cache.forEach(close => close())
```

Comment:
As per https://github.com/HyperCubeProject/binance-api-node/pull/36, I'm thinking it might be interesting to return an object exposing this clean function + the websockets array for advanced users, thoughts?

Reply:
Sure that'll be useful but as I [mentioned before](https://github.com/HyperCubeProject/binance-api-node/issues/24#issuecomment-360649515) the problem with this approach is that it's going to break any apps using the existing api with the close callbacks as this is not backwards compatible. Plus, if the purpose of returning the websocket object back to the caller is so that the caller can handle errors and reconnect by himself then it won't be necessary if reconnection is handled internally by this lib (which I think it should)

Reply:
We can do a major release, that's not a big deal